tests/all/func.rs

```diff
@@ -2139,12 +2152,11 @@ fn call_func_with_funcref_both_typed_and_untyped() -> Result<()> {
     Ok(())
 }
 
-#[test]
+#[wasmtime_test(strategies(not(Winch)))]
```

Comment:
Thought on this, the reason that this test isn't supported on Winch is because it requires wasm gc, right? If that's the case it might be reasonable to have something like `features(gc)` for this macro. The macro would have built-in knowledge that Winch doesn't support `gc` but it would also additionally pre-configure the input `Config` to have `wasm_gc(true)` for example.

That might make things easier to un-ignore for winch in the future as it would only involve changing one location for "ok now winch supports gc" as opposed to hunting down the various exceptions